core/modules/system/system.install

```diff
@@ -403,9 +403,10 @@ function system_requirements($phase) {
         $current_schema = backdrop_get_installed_schema_version($module);
         $latest_schema = end($updates);
         if ($current_schema < $latest_schema) {
+          $update_link = base_path() . 'core/update.php';
           $requirements['update']['severity'] = REQUIREMENT_ERROR;
-          $requirements['update']['value'] = $t('Out of date') . ' (' . l(t('Run database updates'), 'core/update.php'). ')';
-          $requirements['update']['description'] = $t('Some modules have database schema updates to install. You should run the <a href="@update">database update script</a> immediately.', array('@update' => base_path() . 'core/update.php'));
+          $requirements['update']['value'] = $t('Out of date') . ' (<a href="' . $update_link . '">' . t('Run database updates') . '</a>)';
```

Comment:
We're not supposed to use a variable in a string like that.

Instead we do variable substitution via t / format_string. There are some examples in the [docs](https://docs.backdropcms.org/api/backdrop/core%21includes%21bootstrap.inc/function/t/1)

See also right below `<a href="@update">` for an example.

Reply:
@indigoxela I did consider a structure like `
$t('Out of date') . ' (' . t('<a href="@updates">Run database updates</a>', array('@updates' => $update_link)) . ')'
`
but I decided against that because it makes the call to "t" overly complex.  However, if that is what is required ...